libs/libc/time/lib_gmtimer.c

```diff
@@ -51,10 +51,6 @@
  * Pre-processor Definitions
  ****************************************************************************/
```

Comment:
If the "Pre-processor Definitions" section is now empty, then the preceding block comment should also be removed.